### DISCUSSION TITLE
Disable NTFS Last Access Time Stamps

### DISCUSSION THREAD

**Discussion by 1aTa** [Atlas-OS/Atlas#1167] [Suggestions]
I don't personally use Atlas but I've just been through your tweaks and it's a great collection, nice job!

I didn't notice anything related to disabling NTFS last access time stamps but in my experience this is a great performance tweak:

`fsutil behavior set disablelastaccess 1`

or

`[HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem]`
`"NtfsDisableLastAccessUpdate"=dword:80000001`

Another great tweak to make is disabling SmartScreen which can nuke disk performance.

**Comment by nicklvh** [comment]
Pretty sure these are both already done but correct me if I'm wrong

**Comment by 1aTa** [comment]
Ah you're right, last access time stamps is there.

Not sure about SmartScreen.

**Reply by Ast3risk-ops** [reply]
I believe that *is* disabled considering enabling defender enables the bare minimum.

**Reply by 1aTa** [reply]
Yeah SmartScreen is disabled too with NoDefender.

Next time I do a clean install I'll definitely give Atlas a try.
